### ISSUE TITLE
If dependencies aren't installed, the console app shuts down immediately without user interaction.

### ISSUE THREAD

**Issue by winskil** [rfuzzo/CP77Tools#11]
Title says it all. Probably a good idea to post a download link of the dependencies to Nexusmods and/or require user input to close the console app.
![Untitled](https://user-images.githubusercontent.com/44670482/102024661-fb48a780-3d9b-11eb-9da2-cc6d5a0119e1.png)

**Comment by rfuzzo** [comment]
yeah thanks! I need to overhaul the startup anyways, that's a good suggestion!

**Comment by UnboundGodz** [comment]
@winskil Seems an older version of VS like VS2017 doesnt like it due to .NET Framework/Core being outdated, but 2019 will run it fine.

**Comment by martinsstuff** [comment]
> @winskil Seems an older version of VS like VS2017 doesnt like it due to .NET Framework/Core being outdated, but 2019 will run it fine.

I didn't see there was a build available on Nexusmods (a link to there or also uploading builds to the repo's releases might be a good idea) so I just ended up installing the .NET 5.0 SDK and building it via `dotnet msbuild /p:Configuration=Release /restore` which seemed to work without any issues. I actually forgot to copy over `oo2ext_7_win64.dll` but it extracted the .archive with the video files just fine.

**Comment by sodanakin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Github](https://trello.com/c/64mYhNeg/14-github)
